stdlib/2/unittest.pyi

```diff
@@ -15,6 +15,10 @@ _FT = TypeVar('_FT')
 _ExceptionType = Union[Type[BaseException], Tuple[Type[BaseException], ...]]
 _Regexp = Union[Text, Pattern[Text]]
 
+_SysExcInfoType = Tuple[Optional[Type[BaseException]],
+                        Optional[BaseException],
+                        Optional[types.TracebackType]]
```

Comment:
Would it make more sense to use `Union[Tuple[None, None, None], Tuple[Type[BaseException], BaseException, types.TracebackType]]`?

Reply:
You're right, that matches `sys.exc_info()`. I copied it from the Python 3 stubs, I'll change it there too.